src/main/java/frc/robot/commands/DebugValue.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.commands;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.NetworkTableValue;
+
+public class DebugValue {
+
+    public static java.lang.Object Get(String DataName) {
```

Comment:
Our method and variable names follow the Java/Google style guide, camelCase with first word lowercased.